tensorflow_lite_support/python/task/core/task_options.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Options to configure Task APIs."""
+
+import dataclasses
+
+from typing import Optional
+
+
+@dataclasses.dataclass
+class ExternalFile:
+  """Represents external files used by the Task APIs.
+
+  The files can be specified by one of the following
+  two ways:
+
+  (1) file contents loaded in `file_content`.
+  (2) file path in `file_name`.
+
+  If more than one field of these fields is provided, they are used in this
+  precedence order.
+  """
+  # The file contents as a byte array.
+  file_content: Optional[bytearray] = None
+
+  # The path to the file to open and mmap in memory.
+  file_name: Optional[str] = None
+
+
+@dataclasses.dataclass
+class BaseOptions:
```

Comment:
Please reuse the base_option [here](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/python/task/core/proto/base_options.proto). We'll work on migrating existing protos (e.g. base_otpions.proto) to dataclass in a another CL. Let's keep this CL to focus on just NLClassifier.